index.html

```diff
@@ -1049,6 +1049,371 @@ <h2>Evidence</h2>
+				"role": "Project Manager"
+			},
+			"revocation": {
+				"id": "http://example.gov/revocations/738",
+				"type": "SimpleRevocationList2017"
+			},
+			"proof": {
+				"type": "LinkedDataSignature2015",
+				"created": "2016-06-18T21:19:10Z",
+				"creator": "https://example.com/jdoe/keys/1",
+				"domain": "json-ld.org",
+				"nonce": "598c63d6",
+				"signatureValue": "BavEll0...W3JT24 = "
+			}
+		}
+	},
+	"proof": { ...
+	}
+}
+
+        </pre>
+
+
+  </section>
+
+  <section>
+   <h3> Delegation through Profiles </h3>
+
+   <p>
+
+Through the use of profiles, the subject delegates a credential to a
+holder, by issuing a new credential in which: the issuer is the subject, 
+the subject is the delegated holder, and the claim contains the property to be delegated. In addition, the delegated holder creates a profile that contains these two credentials, and a relationships property in which one
+relationship refers to the original subject and has the value 'delegate' and the other
+relationship refers to the delegated credential and has the value 'self'.
+
+        <pre class="example nohighlight" title="An example of delegation using a profile">
+{
+	"id": "did:example:ebfeb1c276e12ec211f712ebc6f",
+	"type": ["VerifiableProfile"],
+	"holderid": "did:example:76e12ec21ebhyu1f712ebc6f1z2",
+	"relationships": [{
+			"subjectid": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+			"relationship": "delegate"
+		},
+		{
+			"subjectid": "did:example:76e12ec21ebhyu1f712ebc6f1z2",
+			"relationship": "self"
+		}
+	],
+	"credential": [{
+			"id": "http://example.gov/credentials/3732",
+			"type": ["Credential", "Role"],
+			"issuer": "https://dmv.example.gov",
+			"issued": "2010-01-01",
+			"claim": {
+				"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+				"role": "Project Manager"
+			},
+			"revocation": {
+				"id": "http://example.gov/revocations/738",
+				"type": "SimpleRevocationList2017"
+			},
+			"proof": {....}
+		},
+		{
+			"id": "some other ID",
+			"type": ["Credential", "DelagatedCredential"],
+			"issuer": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+			"issued": "2010-01-03",
+			"claim": {
+				"id": "did:example:76e12ec21ebhyu1f712ebc6f1z2",
+				"role": "Project Manager"
+			},
+			"proof": {
+				"type": "RsaSignature2018",
+				"created": "2017-06-17T10:03:48Z",
+				"creator": "did:example:ebfeb1f712ebc6f1c276e12ec21/keys/234",
+				"nonce": "d61c4599-0cc2-4479-9efc-c63add3a43b2",
+				"signatureValue": "pYw8XNi1..Cky6Ed = "
+			}
+		}
+	],
+	"proof": [{
+		"type": "RsaSignature2018",
+		"created": "2017-06-18T21:19:10Z",
+		"creator": "did:example:76e12ec21ebhyu1f712ebc6f1z2/keys/2",
+		"nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
+		"signatureValue": "BavEll0/I1..W3JT24 = "
+	}]
+}
+        </pre>
+
+
+   </section>
+  </section>
+
+
+  <section>
+      <h2>Issuer Authorises Holder</h2>
+
+      <p>
+When the issuer wishes to authorise a holder to possess a credential that 
```

Comment:
This is a good idea. I like it, and it simplifies the task of the verifier, since it only needs to trust the issuer and not rely on the holder. Of course it now places the burden on the issuer instead of the verifier, but that's OK since the issuer needs to have a good reason for giving the VC to a holder who is not the subject.